### ISSUE TITLE
Upgrade Vagrantfile to use the latest Ubuntu LTS

### ISSUE THREAD

**Issue by matiashf** [hioa-cs/IncludeOS#530]
The IncludeOS Vagrantfile currently builds with Ubuntu 14.04, but the latest is 16.04.

**Comment by matiashf** [comment]
I changed my Vagrantfile like so:
```
    config.vm.box = "xenial-server-cloudimg-amd64" # Basename of the url below
    config.vm.box_url = "https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-vagrant.box"
```

However, when running `vagrant up` I get the following error message after a _long_ time-out:
```
==> IncludeOS: Mounting shared folders...
    IncludeOS: /vagrant => /home/mhf/Sync/gits/IncludeOS
Failed to mount folders in Linux guest. This is usually because
the "vboxsf" file system is not available. Please verify that
the guest additions are properly installed in the guest and
can work properly. The command attempted was:

mount -t vboxsf -o uid=`id -u ubuntu`,gid=`getent group ubuntu | cut -d: -f3` vagrant /vagrant
mount -t vboxsf -o uid=`id -u ubuntu`,gid=`id -g ubuntu` vagrant /vagrant

The error output from the last command was:

sudo: unable to resolve host ubuntu-xenial
mesg: ttyname failed: Inappropriate ioctl for device
mount: unknown filesystem type 'vboxsf'
```

This means that guest additions are not installed by default in the latest LTS. When I install `virtualbox-guest-utils` on the virtual machine, mounting proceeds nicely. However, it seems that Vagrant will attempt to mount shared folders before executing shell commands, so we can't simply include the install command as part of the Vagrantfile because it will never be executed.

It seems to me that we have to either figure out some way to make vagrant install guest additions on the host before attempting to mount the shared directory, or build (and maintain) our own ubuntu image with guest additions installed by default.

**Comment by ferrarimarco** [comment]
Hi, you may want to have a look at this: https://bugs.launchpad.net/cloud-images/+bug/1565985

**Comment by xwvvvvwx** [comment]
Fixed by #928. This can be closed :)
